src/main/kotlin/net/maizegenetics/phgv2/brapi/api/Samples.kt

```diff
@@ -4,10 +4,10 @@ import io.ktor.http.*
 import io.ktor.server.application.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
-import model.Metadata
 import net.maizegenetics.phgv2.brapi.model.SampleListResponse
 import net.maizegenetics.phgv2.brapi.model.SampleListResponseResult
 import net.maizegenetics.phgv2.brapi.service.SamplesService
+import net.maizegenetics.phgv2.brapi.model.Metadata
```

Comment:
Still confusing to me why an import only would be needed here. Didn't it compile before?

Reply:
When I remove the import, it thinks "Metadata is "kotlin.Metadata" instead of the brapi.model.Metadata.

i'll look at main and see what is there to understand why it did or didn't need this import.

Reply:
@tcasstevens branch main has the import "import model.Metadata", which was the wrong package, but it did appear to need an import.  The new code has the import with the correct/updated package name.